proto/buf/alpha/registry/v1alpha1/studio_request.proto

```diff
@@ -0,0 +1,141 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+import "google/protobuf/timestamp.proto";
+
+// StudioRequestService exposes the functionality to manage favorited Requests
+// from Studio.
+service StudioRequestService {
+  // CreateStudioRequest registers a favorite Studio Requests to the caller's
+  // BSR profile.
+  rpc CreateStudioRequest(CreateStudioRequestRequest) returns (CreateStudioRequestResponse) {}
+  // RenameStudioRequest renames an existing Studio Request.
+  rpc RenameStudioRequest(RenameStudioRequestRequest) returns (RenameStudioRequestResponse) {}
```

Comment:
@paul-sachs @doriable as discussed in the zoom, we're not adding a full capability to update the favorite, but just rename it. For edits, user can just un-favorite and favorite a new one.

Reply:
Sure, as long as we're not making any decisions that makes it impossible to create a full edit endpoint. I can def imagine that we'll want edit functionality at some point, most MVP.